pom.xml

```
@@ -51,8 +51,8 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.249.x</artifactId>
-        <version>984.vb5eaac999a7e</version>
+        <artifactId>bom-weekly</artifactId>
```

Weekly should only be used if you are actively tracking the latest LTS version.
Otherwise the closest LTS line should be used

```suggestion
        <artifactId>bom-2.319.x</artifactId>
```

---

Rather switch to https://github.com/jenkinsci/bom/pull/876 I guess.

---

319 should be fine for now 😄 

can be bumped to next LTS separately (also fine to bump to next LTS but need to get the `ldap` plugin tests sorted first).

---

https://github.com/jenkinsci/bom/releases/tag/1155.v77b_fd92a_26fc is ready.

---

@car-roll I would suggest as the maintainer just commit whatever you feel like here 😄